pkg/api-server/resource_endpoints.go

```diff
@@ -424,7 +424,9 @@ func (r *resourceEndpoints) createResource(
 			log.Error(err, "Could not write the response")
 		}
 	} else {
-		response.WriteHeader(201)
+		if err := response.WriteHeaderAndJson(201, struct{}{}, "application/json"); err != nil {
```

Comment:
there are constants `http.StatusOK` and `http.StatusCreated` we're using them in other places

Reply:
done